ERROR	2022-07-29 14:55:13,888	0	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 17de03da9f99eacd2dee17441eb25ecc93f86b7f0fbec4761bea737e00eef1b4 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 14:55:13,892	4	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 17de03da9f99eacd2dee17441eb25ecc93f86b7f0fbec4761bea737e00eef1b4 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 17de03da9f99eacd2dee17441eb25ecc93f86b7f0fbec4761bea737e00eef1b4 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 14:58:48,934	0	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: af797d2a3a5d91039e1af8944c6ea498d80d3b6ce33ef57870b09b48e006ebb8 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 14:58:48,939	5	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: af797d2a3a5d91039e1af8944c6ea498d80d3b6ce33ef57870b09b48e006ebb8 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: af797d2a3a5d91039e1af8944c6ea498d80d3b6ce33ef57870b09b48e006ebb8 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 15:00:31,439	0	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d9e6a6edd68fec3e21430cc34f56a47a0f7bbdaba5f3ceff2b0bd24e29a383c1 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 15:00:31,443	4	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d9e6a6edd68fec3e21430cc34f56a47a0f7bbdaba5f3ceff2b0bd24e29a383c1 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d9e6a6edd68fec3e21430cc34f56a47a0f7bbdaba5f3ceff2b0bd24e29a383c1 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 15:12:40,483	0	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: e884a3a210dc6e5678ffd3e5f4d115e089796a5a4ca54acad1cc0d5bc9efd752 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 15:12:40,536	53	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: e884a3a210dc6e5678ffd3e5f4d115e089796a5a4ca54acad1cc0d5bc9efd752 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: e884a3a210dc6e5678ffd3e5f4d115e089796a5a4ca54acad1cc0d5bc9efd752 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:55)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 15:14:08,976	1	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 5129fb698ec5db9f49320a9489651e8a8631ac81f9dc5eccd5d01cadfa35f51d to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 15:14:08,980	5	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 5129fb698ec5db9f49320a9489651e8a8631ac81f9dc5eccd5d01cadfa35f51d to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: 5129fb698ec5db9f49320a9489651e8a8631ac81f9dc5eccd5d01cadfa35f51d to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 16:05:16,849	1	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d0fb19ca1c0b946f784f0013ef8355c8ec444a981c954c0a1916403a50d2498c to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:05:16,874	26	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d0fb19ca1c0b946f784f0013ef8355c8ec444a981c954c0a1916403a50d2498c to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d0fb19ca1c0b946f784f0013ef8355c8ec444a981c954c0a1916403a50d2498c to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 16:06:19,735	0	org.hyperledger.fabric.sdk.Channel	[main]	Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d892cc7c8f0fc760075f54956f9d7b4f3199afc59500eee4fdcbc2783530db44 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
java.lang.Exception: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4965)
	at org.hyperledger.fabric.sdk.Channel.sendProposalToPeers(Channel.java:4868)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:941)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
Caused by: io.grpc.StatusRuntimeException: UNKNOWN: error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$UnaryStreamToFuture.onClose(ClientCalls.java:490)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusStatsModule$StatsClientInterceptor$1$1.onClose(CensusStatsModule.java:700)
	at io.grpc.PartialForwardingClientCallListener.onClose(PartialForwardingClientCallListener.java:39)
	at io.grpc.ForwardingClientCallListener.onClose(ForwardingClientCallListener.java:23)
	at io.grpc.ForwardingClientCallListener$SimpleForwardingClientCallListener.onClose(ForwardingClientCallListener.java:40)
	at io.grpc.internal.CensusTracingModule$TracingClientInterceptor$1$1.onClose(CensusTracingModule.java:399)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:510)
	at io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:630)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$700(ClientCallImpl.java:518)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:692)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:681)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:06:19,739	4	org.hyperledger.fabric.sdk.Channel	[main]	getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d892cc7c8f0fc760075f54956f9d7b4f3199afc59500eee4fdcbc2783530db44 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
org.hyperledger.fabric.sdk.exception.ProposalException: getConfigBlock for channel mychannel failed with peer peer0.org1.example.com.  Status FAILURE, details: Channel Channel{id: 1, name: mychannel} Sending proposal with transaction: d892cc7c8f0fc760075f54956f9d7b4f3199afc59500eee4fdcbc2783530db44 to Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} failed because of: gRPC failure=Status{code=UNKNOWN, description=error validating proposal: access denied: channel [mychannel] creator org unknown, creator is malformed, cause=null}
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:952)
	at org.hyperledger.fabric.sdk.Channel.getConfigBlock(Channel.java:907)
	at org.hyperledger.fabric.sdk.Channel.parseConfigBlock(Channel.java:1994)
	at org.hyperledger.fabric.sdk.Channel.loadCACertificates(Channel.java:1831)
	at org.hyperledger.fabric.sdk.Channel.initialize(Channel.java:1222)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.initializeChannel(NetworkImpl.java:59)
	at org.hyperledger.fabric.gateway.impl.NetworkImpl.<init>(NetworkImpl.java:50)
	at org.hyperledger.fabric.gateway.impl.GatewayImpl.getNetwork(GatewayImpl.java:252)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:56)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.<init>(ConformativeGame.java:29)
	at com.dialectek.conformative.hyperledger.client.ConformativeGame.main(ConformativeGame.java:156)
ERROR	2022-07-29 16:24:10,248	1	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLEec2ed1c9-d583-43c1-b7ed-e18e71521a16BLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:25:47,038	0	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE7459fe0b-1d9e-440f-bb92-c0393824bcfbBLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:25:47,041	3	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE56aaa1e5-fa35-4c40-9be6-99850ed3b4eeBLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:43:06,065	1	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE82e58a96-d0b8-477b-b7dd-b843680a7975BLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 5, name: peer0.org2.example.com:9051, channelName: mychannel, url: grpcs://localhost:9051, mspid: Org2MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:43:06,067	3	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE541b09d3-10e8-4d0f-98b4-23d08731b946BLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 5, name: peer0.org2.example.com:9051, channelName: mychannel, url: grpcs://localhost:9051, mspid: Org2MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:48:22,475	0	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE8087a749-4c8d-4ec1-8ae5-e052c83ff1baBLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 5, name: peer0.org2.example.com:9051, channelName: mychannel, url: grpcs://localhost:9051, mspid: Org2MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:54:53,402	0	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLEb033448a-5a18-4823-bc3b-ea73587d3226BLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 5, name: peer0.org2.example.com:9051, channelName: mychannel, url: grpcs://localhost:9051, mspid: Org2MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-29 16:56:05,917	0	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE0887121c-4afd-46e0-8dc8-736c952af39eBLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 5 for channel: mychannel, from Peer{ id: 5, name: peer0.org2.example.com:9051, channelName: mychannel, url: grpcs://localhost:9051, mspid: Org2MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-30 10:51:26,685	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 10:51:26,697	12	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 10:53:50,827	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 10:53:50,829	2	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 10:53:52,612	1785	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE2120046a-66b0-44bf-a822-e7ebe6d03c0bBLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 6 for channel: mychannel, from Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-30 10:53:52,614	1787	org.hyperledger.fabric.sdk.Channel	[pool-1-thread-1]	Error calling block listener BLOCK_LISTENER_HANDLE0a6762ff-12b0-4e66-9b31-fcc88ba812d9BLOCK_LISTENER_HANDLE on channel: mychannel event: Channel mychannel eventqueue got block event with block number: 6 for channel: mychannel, from Peer{ id: 2, name: peer0.org1.example.com, channelName: mychannel, url: grpcs://localhost:7051, mspid: Org1MSP} 
java.lang.NullPointerException
	at org.hyperledger.fabric.sdk.Channel.lambda$startEventQue$13(Channel.java:5974)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
ERROR	2022-07-30 12:10:29,841	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:10:29,842	1	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 12:12:51,240	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:12:51,242	2	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 12:22:58,514	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:22:58,530	16	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 12:23:59,405	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:23:59,406	1	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@71075444
ERROR	2022-07-30 12:26:10,027	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:26:10,028	1	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 12:34:22,897	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	POST request to https://localhost:7054 failed request body {"id":"appUser","type":"client","affiliation":"org1.department1","attrs":[]}. Response: {"result":"","errors":[{"code":74,"message":"Identity 'appUser' is already registered"}
],"messages":[],"success":false}
ERROR	2022-07-30 12:34:22,898	1	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	com.dialectek.conformative.hyperledger.client.NetworkClient$1@645aa696
ERROR	2022-07-30 13:51:03,891	1	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	Connect to localhost:7054 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
org.apache.http.conn.HttpHostConnectException: Connect to localhost:7054 [localhost/127.0.0.1] failed: Connection refused (Connection refused)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:156)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:376)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at org.hyperledger.fabric_ca.sdk.HFCAClient.httpPost(HFCAClient.java:1326)
	at org.hyperledger.fabric_ca.sdk.HFCAClient.enroll(HFCAClient.java:478)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.enrollAdmin(NetworkClient.java:128)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.init(NetworkClient.java:48)
	at com.dialectek.conformative.hyperledger.client.NetworkClient.main(NetworkClient.java:216)
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:607)
	at org.apache.http.conn.ssl.SSLSocketFactory.connectSocket(SSLSocketFactory.java:543)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
	... 14 more
ERROR	2022-07-31 13:58:11,661	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	org.hyperledger.fabric_ca.sdk.exception.InvalidArgumentException: Unable to add CA certificate, can't open certificate file /home/tom/Documents/research/blockchain/Hyperledger/Fabric/dapps/ConformativeChain/conformative-client/../../../organizations/peerOrganizations/org1.example.com/ca/ca.org1.example.com-cert.pem
ERROR	2022-07-31 13:59:09,855	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	org.hyperledger.fabric_ca.sdk.exception.InvalidArgumentException: Unable to add CA certificate, can't open certificate file /home/tom/Documents/research/blockchain/Hyperledger/Fabric/dapps/ConformativeChain/conformative-client/../../../organizations/peerOrganizations/org1.example.com/ca/ca.org1.example.com-cert.pem
ERROR	2022-07-31 14:00:45,205	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	org.hyperledger.fabric_ca.sdk.exception.InvalidArgumentException: Unable to add CA certificate, can't open certificate file /home/tom/Documents/research/blockchain/Hyperledger/Fabric/dapps/ConformativeChain/conformative-client/../../../organizations/peerOrganizations/org1.example.com/ca/ca.org1.example.com-cert.pem
ERROR	2022-07-31 14:03:41,536	0	org.hyperledger.fabric_ca.sdk.HFCAClient	[main]	org.hyperledger.fabric_ca.sdk.exception.InvalidArgumentException: Unable to add CA certificate, can't open certificate file /home/tom/Documents/research/blockchain/Hyperledger/Fabric/dapps/ConformativeChain/conformative-client/../../../organizations/peerOrganizations/org1.example.com/ca/ca.org1.example.com-cert.pem
